ynab_to_ledger.rb

```diff
@@ -35,7 +35,7 @@ def ledger_entry(row)
 end
 
 def blank_if_zero(amount)
-  amount =~ /\A\$?0.00\z/ ? "" : amount
+  amount =~ /\A\$?0(\.0+)?\z/ ? "" : amount
```

Comment:
Would this be simpler? `/\A\$?0\.0+\z/`?

I think it's probably time to add some tests and we can cover both my file and your file.

Reply:
My regex also works for a simple `0` without a decimal point. I imagine there are some currencies that don't use the decimal point. Good idea about the tests.

Reply:
Makes sense. I'll work on some tests. Thanks!